docs/ashell.md

```diff
@@ -183,6 +184,15 @@ Renames filename1 to filename2.
 
 Clears the terminal screen.
 
+### erase
+
+`erase`
+
+Erase flash file system.
+
+Note: this only works on the Arduino 101 and it will require rebooting the
+device to take affect.
```

Comment:
affect -> effect

What do you mean by this? It looks like it happens right away; do you mean that you think their view of filesystem may be inconsistent after the operation if they don't reboot? If so, how about we reboot for them...